### ISSUE TITLE
[CLOSED] Include labels/probabilities in inference script outputs.

### ISSUE THREAD

**Issue by jeswan** [nyu-mll/jiant-v1-legacy#507]
<a href="https://github.com/sleepinyourhat"><img src="https://avatars1.githubusercontent.com/u/1284441?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [sleepinyourhat](https://github.com/sleepinyourhat)**
_Wednesday Mar 06, 2019 at 17:57 GMT_
_Originally opened as https://github.com/nyu-mll/jiant/issues/507_

----

Format the output from the REPL in a way that's easier to evaluate manually: 
Show predicted labels and probabilities in addition to logits.
